diff --git a/component/outbound/outbound.go b/component/outbound/outbound.go
index bc16fb0..902525c 100644
--- a/component/outbound/outbound.go
+++ b/component/outbound/outbound.go
@@ -10,6 +10,7 @@ import (
 	_ "github.com/daeuniverse/outbound/dialer/http"
 	_ "github.com/daeuniverse/outbound/dialer/hysteria2"
 	_ "github.com/daeuniverse/outbound/dialer/juicity"
+	_ "github.com/daeuniverse/outbound/dialer/masque"
 	_ "github.com/daeuniverse/outbound/dialer/naive"
 	_ "github.com/daeuniverse/outbound/dialer/shadowsocks"
 	_ "github.com/daeuniverse/outbound/dialer/shadowsocksr"
